### PR TITLE
connected frontend w/ backend

### PR DESCRIPTION
**Pull request by ashabanov-exceed**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -133,33 +153,69 @@ render = () => {
...
+    },
+    body: JSON.stringify({
+      id: allTasks[index].id,
+      text: changedInput.value,
+      isCheck: index.isCheck
+    })
+  });
+  let result = await resp.json();
+  allTasks = result.data;
+  // localStorage.setItem('task', JSON.stringify(allTasks));
```
Comment (ashabanov-exceed) [inline script.js:218]: will be removed with new commit

File: file%3a/home/user/Documents/module%25203/to-do-gh/style.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+.button-div {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 35%;
+  justify-content: center;
+}
+
+.button-div {}
```
Comment (usatovaDasha) [inline file%3a/home/user/Documents/module%25203/to-do-gh/style.css:154]: empty class?

File: script.js [hunk 3]
```diff
@@ -4,21 +4,41 @@ let input = null;
 let indexEdit = -1;
 let valueChange = "";
 
-window.onload = function init() {
+window.onload = async function init() {
```
Comment (usatovaDasha) [inline script.js:7]: es6 function

File: script.js [hunk 4]
```diff
@@ -133,33 +153,69 @@ render = () => {
 
 };
 
-onChangeCheckbox = (index) => {
+onChangeCheckbox = async (index) => {
   allTasks[index].isCheck = !allTasks[index].isCheck;
-  localStorage.setItem('task', JSON.stringify(allTasks));
+
+  console.log(allTasks[index].id)
```
Comment (usatovaDasha) [inline script.js:159]: remove

File: script.js [hunk 5]
```diff
@@ -133,33 +153,69 @@ render = () => {
...
+  const resp = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+      'Access-Control-Allow-Origin': "*"
+    },
+    body: JSON.stringify({
+      id: allTasks[index].id,
```
Comment (usatovaDasha) [inline script.js:167]: destructurization

File: script.js [hunk 6]
```diff
@@ -133,33 +153,69 @@ render = () => {
...
+  let result = await resp.json();
+  allTasks = result.data;
   render();
 };
 
-onClickRemove = (index) => {
-  allTasks.splice(index, 1);
-  localStorage.setItem('task', JSON.stringify(allTasks));
+onClickRemove = async (index) => {
+  const resp = await fetch(`http://localhost:8000/deleteTask?id=${allTasks[index].id}`, {
```
Comment (usatovaDasha) [inline script.js:178]: destructurization

File: script.js [hunk 7]
```diff
@@ -133,33 +153,69 @@ render = () => {
...
+  const resp = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+      'Access-Control-Allow-Origin': "*"
+    },
+    body: JSON.stringify({
+      id: allTasks[index].id,
+      text: changedInput.value,
+      isCheck: index.isCheck
```
Comment (usatovaDasha) [inline script.js:213]: index?

File: script.js [hunk 8]
```diff
@@ -133,33 +153,69 @@ render = () => {
...
   indexEdit = -1;
-  localStorage.setItem('task', JSON.stringify(allTasks));
+  const resp = await fetch('http://localhost:8000/updateTask', {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+      'Access-Control-Allow-Origin': "*"
+    },
+    body: JSON.stringify({
+      id: allTasks[index].id,
```
Comment (usatovaDasha) [inline script.js:211]: destructurization

**Pull request merged**